main.go

```diff
@@ -44,7 +63,7 @@ func main() {
 	flagset.StringVar(&label, "label", "", "The label to enforce in all proxied PromQL queries. "+
 		"This label will be also required as the URL parameter to get the value to be injected. For example: -label=tenant will"+
 		" make it required for this proxy to have URL in form of: <URL>?tenant=abc&other_params...")
-	flagset.StringVar(&labelValue, "label-value", "", "A fixed label value to enforce in all proxied PromQL queries. "+
+	flagset.Var(&labelValue, "label-value", "A fixed label value to enforce in all proxied PromQL queries. "+
```

Comment:
```suggestion
	flagset.Var(&labelValue, "label-value", "A fixed label value to enforce in all proxied PromQL queries. It can be repeated in which case the proxy will enforce the union of values. "+
```